### DISCUSSION TITLE
Update package.json of one package when another package's version changes.

### DISCUSSION THREAD

**Discussion by rwacarter** [changesets/changesets#1103] [Q&A]
In a monorepo, let's say I have PackageA and PackageB. PackageB is a dependency of PackageA, with a line in the PackageA package.json similar to: `"packageb": "^1.0.0"`. Let's say I then use Changesets to update the version of PackageB to `1.0.1`. Is there a way to automatically update the line in the PackageA package.json to reflect this change in version?

**Comment by Andarist** [comment]
Could you try this one?
```json5
// .changeset/config.json
{
  "updateInternalDependencies": "patch"
}
```

alternatively, there is a chance that this would match your expectations:
```json5
// .changeset/config.json
{
  "___experimentalUnsafeOptions_WILL_CHANGE_IN_PATCH": {
    "updateInternalDependents": "always"
  }
}
```

**Reply by karol-f** [reply]
Thank you! This `___experimentalUnsafeOptions_WILL_CHANGE_IN_PATCH` works like a charm! 

Now I can have NPM publishable libraries with e.g. caret version of monorepo dependency in package.json and it always updates it to the latest caret version of it. Great, thank you for the tip!
